ovh/types_cloud_project_database.go

```diff
@@ -71,6 +73,7 @@ func (v CloudProjectDatabaseResponse) ToMap() map[string]interface{} {
 	obj["plan"] = v.Plan
 	obj["status"] = v.Status
 	obj["version"] = v.Version
+	obj["disk"] = v.Disk.ToMap()
```

Comment:
this should be align with schema because that will be set in terraform state an compare with the schema:
```go
obj["disk_size"] = v.Disk.Size
obj["disk_type"] = v.Disk.Type // This need to add disk_type as computed attribut in the schema
```